src/neo/Cryptography/MPT/MPTTrie.cs

```diff
@@ -8,17 +8,18 @@ public partial class MPTTrie<TKey, TValue>
         where TKey : notnull, ISerializable, new()
         where TValue : class, ISerializable, new()
     {
-        private const byte Prefix = 0xf0;
+        private byte Prefix;
```

Comment:
Why this is changed?

Reply:
If we have multiple MPTs, then `Prefix` need to be passed by the outside. And we have defined all the prefixes of table in `Prefixes.cs` file.

Reply:
What is the benefit using hard code.

Reply:
> If we have multiple MPTs

Why do we need multiple MPTs?

Reply:
As neo-core is the library, some other clients may want to use the MPT.

Reply:
Why does a user care about what prefix should be used? I think it is a implementation detail. A caller care about `key`, `value`, but not `prefix`.

Reply:
okay.